rubocop-shopify.gemspec

```
@@ -21,5 +21,5 @@ Gem::Specification.new do |s|
     "allowed_push_host" => "https://rubygems.org",
   }
 
-  s.add_dependency("rubocop", ">= 0.85", "< 0.89")
+  s.add_dependency("rubocop", "0.89")
```

```suggestion
  s.add_dependency("rubocop", "~> 0.89.0")
```

We need to allow patches for this version.